src/ebpf/rs/ebpf_trigger.rs

```diff
@@ -23,6 +23,9 @@ pub struct ProcessStartTrigger {
     pub file_name: String,
```

Comment:
can you also remove this filename? i don't think it's useful and we never get this?